rbx_xml/src/tests/snapshots/rbx_xml__tests__basic__migrated_properties.snap

```diff
@@ -0,0 +1,65 @@
+---
+source: rbx_xml/src/tests/basic.rs
+expression: "std::str::from_utf8(&encoded).unwrap()"
+---
+<roblox version="4">
+  <Item class="Folder" referent="0">
+    <Properties>
+      <string name="Name">Folder</string>
+    </Properties>
+    <Item class="ScreenGui" referent="1">
+      <Properties>
+        <string name="Name">ScreenGui</string>
+        <token name="ScreenInsets">0</token>
+      </Properties>
+    </Item>
+    <Item class="ScreenGui" referent="2">
+      <Properties>
+        <string name="Name">ScreenGui</string>
+        <token name="ScreenInsets">1</token>
+      </Properties>
+    </Item>
+    <Item class="Part" referent="3">
+      <Properties>
+        <string name="Name">Part</string>
+        <Color3uint8 name="Color3uint8">16777215</Color3uint8>
+      </Properties>
+    </Item>
+    <Item class="Part" referent="4">
+      <Properties>
+        <string name="Name">Part</string>
+        <Color3uint8 name="Color">11829503</Color3uint8>
```

Comment:
Maybe the new property for `PropertyMigration::BrickColorToColor` should really be `Color3uint8`? 🤔 It'll still work regardless though...

Reply:
I suspect yes, it should be. With the way Roblox handles reflection, it turns out to be fine, but we should probably fix that for correctness's sake.